src/bcf/report/oncoprint.rs

```diff
@@ -373,12 +373,12 @@ pub fn oncoprint(
                     1 => rec.extend(filter_canonical.iter().map(|(r, _, _)| r.clone())),
                     _ => {
                         rec.extend(filter_canonical.iter().map(|(r, _, _)| r.clone()));
-                        let alterations = filter_canonical
+                        let transcripts = filter_canonical
                             .iter()
                             .map(|(r, _, _)| r.alteration.clone())
```

Comment:
Why is this attribute called alteration?

Reply:
Well i think because it simply is. This contains `ENSP00000378284.3:p.Glu436Gly` for example. At this stage only the alteration, the sample and the gene are present (see struct `SecondStageRecord`) because that is everything thats needed for the second stage of the vcf-report. The whole transcript is only handled from around line 138 to 361.

Reply:
Oh I see. Then, let me suggest another change below.